musicbot/bot.py

```diff
@@ -1635,6 +1664,39 @@ def _gen_embed(self):
 
         return Response(reply_text, delete_after=30)
 
+    async def cmd_mpaths(self,server,author, leftover_args, path = '.'):
+        """
+        Usage:
+            {command_prefix}mpaths path
+
+        Queries folders for local music
+        """
+        songs = u''
+        baseList = self.config.musicPaths.split(",")
+        
```

Comment:
Add a check for if the user gave no input (returns a `FileNotFoundError` on both Windows and Linux)

```125233822760566784/MattBSG#1043: ~mpaths
[ERROR:bot] Exception in on_message
Traceback (most recent call last):
  File "C:\Users\Matthew\Desktop\MusicBot\musicbot\bot.py", line 2834, in on_message
    response = await handler(**handler_kwargs)
  File "C:\Users\Matthew\Desktop\MusicBot\musicbot\bot.py", line 1682, in cmd_mpaths
    fileList = os.listdir(fullPath)
FileNotFoundError: [WinError 3] The system cannot find the path specified: ''```